visualc/VS2010/gen_random_havege.vcxproj

```diff
@@ -165,7 +165,7 @@
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <OptimizeReferences>true</OptimizeReferences>
       <AdditionalLibraryDirectories>Release</AdditionalLibraryDirectories>
-      <AdditionalDependencies>%(AdditionalDependencies);</AdditionalDependencies>
+      <AdditionalDependencies>bcrypt.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
Should we leave this variable unexpanded? Here and in other similar places.

Reply:
What I've done in the most recent commit is to copy the change that was present in the other passing configurations such as 'Release|Win32' into 'Release|x64' which was failing. In a way - I can't say I fully understand the configuration file itself, and have just copied what works elsewhere to what doesn't work.

Given we don't fully understand it, and we generate these files, so have to adjust a template not the files themselves, I'd prefer to be conservative and change only what we have to.

Reply:
that would work, it may have been an oversight as the IDE normally fills in that particular list of import archives